CONTRIBUTING.md

```diff
@@ -6,14 +6,26 @@ Vielen Dank, dass Du bei OctoAwesome mitmachen möchtest und ein Teil davon werd
 
 Bitte beachte bei allem immer unseren [Code of Conduct](CODE_OF_CONDUCT.md), damit wir gemeinsam friedlich zusammenarbeiten können.
 
-<!-- ## FAQ's
-
-Häufig Gestellte fragen, findest Du auf dem GitHub Wiki oder im octoawesome wiki -->
-
 ## Coding
 
-Wenn Du mit programmieren möchtest, beachte bitte unsere Richtlinien zum [Codestyle](http://wiki.octoawesome.net/wiki/Coding_Styles) und unseren [Code of Conduct](CODE_OF_CONDUCT.md). Das heißt, wenn Du größere Änderungen im Code machst, produziere dazu ein Video, einen Stream oder einen Blog beitrag.
+Wenn Du mit programmieren möchtest, beachte bitte unsere Richtlinien zum [Codestyle](http://wiki.octoawesome.net/wiki/Coding_Styles) und unseren [Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## Wiki
 
-Natürlich kann man nicht nur durch Programmieren mithelfen, sondern auch am [Wiki](http://wiki.octoawesome.net/wiki/Hauptseite) mitschreiben. Beachte hierzu bitte unsere [Wiki Richtlinien](http://wiki.octoawesome.net/wiki/Styleguide).
\ No newline at end of file
+Natürlich kann man nicht nur durch Programmieren mithelfen, sondern auch am [Wiki](http://wiki.octoawesome.net/wiki/Hauptseite) mitschreiben. Beachte hierzu bitte unsere [Wiki Richtlinien](http://wiki.octoawesome.net/wiki/Styleguide).
+
+# OctoAwesome Credo
+
+__Das Projekt ist deutschsprachig aber nicht nur__ Ja wir sind eine deutsche Community und wir werden auf Deutsch streamen, deutsche Texte bereitstellen und eine deutsche Doku zur Verfügung stellen. Wir sind aber uns darüber im Klaren, dass die übliche Programmiersprache Englisch ist. Und wir wissen um den Effekt der Muttersprache, wenn es um das lernen geht. Deutsch werden wir als primäre Sprache beibehalten und fördern dennoch möchten wir dafür sorge tragen, dass niemand anderes ausgeschlossen ist, nur weil er eine andere Sprache spricht. Daher möchten wir das alles was im Code passiert (Summary's, Code & Commits) auf Englisch halten, während das Wiki und unsere Diskussionen hier auf GitHub auf Deutsch halten.
+
+__Tue gutes und Rede darüber__ Der Ansatz von OctoAwesome ist die Nachvollziehbarkeit und Transparenz. Ja selbst ein Googeln, eine Idee oder andere kleine Dinge können andere Bereichern. Im Sinne der Community, die von diesem Motto lebt, wünschen wir uns das wenn du etwas für OctoAwesome machst, dies festhälst. Das kann ein Video, Vortrag, Stream oder Blogeintrag sein es gibt wahrlich keine Grenzen. Auch wenn es nicht perfekt ist oder du in eine Sackgasse rennst, andere können davon lernen und auch du selbst kannst davon profitieren. Jedoch ist es nicht erforderlich für eine Mitarbeit am Projekt.
```

Comment:
```suggestion
__Tue gutes und Rede darüber__ Der Ansatz von OctoAwesome ist die Nachvollziehbarkeit und Transparenz. Ja selbst ein Googeln, eine Idee oder andere kleine Dinge können andere bereichern. Im Sinne der Community, die von diesem Motto lebt, wünschen wir uns das wenn du etwas für OctoAwesome machst, dies festhälst. Das kann ein Video, Vortrag, Stream oder Blogeintrag sein es gibt wahrlich keine Grenzen. Auch wenn es nicht perfekt ist oder du in eine Sackgasse rennst, andere können davon lernen und auch du selbst kannst davon profitieren. Jedoch ist es nicht erforderlich für eine Mitarbeit am Projekt.
```

Reply:
```suggestion
__Tue gutes und Rede darüber__ Größere Änderungen an der Code Basis können ohne Wiki Eintrag, Video, Blogeintrag oder ähnliches umgesetzt werden. Natürlich würden wir uns trotzdem freuen, wenn du die Zeit dazu findest einen Öffentlichen Beitrag zu erstellen, damit auch jetzige und zukünftige Mitglieder dieser Community von deinem Wissen profitieren und aus deinen Fehlern lernen können.
```

Reply:
Ich finde, so klingt es weniger als ein Scheitern. Wir sind damit ja nicht gescheitert.

Reply:
Ich find in dem aktuellen zustand saschas variante besser, da es mMn eindeutiger ist.